lib/dialyzer/test/options3_SUITE_data/dialyzer_options

```diff
@@ -0,0 +1 @@
+{dialyzer_options, [{use_spec, false}]}.
```

Comment:
not sure if we can replace the `use_spec` by `no_spec`. 
The main reason is to make the use of these flags as close as possible to the CLI flags.
It is an idea to think about :)

Reply:
`use_spec` does not seem to be documented, so we should be able to change it.

Reply:
The Erlang API is under-documented with only the most important options included in the spec.

I will not address that in this PR. I will keep `use_spec`, but add `no_spec` as a synonym.